resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtils.scala

```diff
@@ -43,7 +43,7 @@ private[spark] object KubernetesClientUtils extends Logging {
 
   val configMapNameExecutor: String = configMapName(s"spark-exec-${KubernetesUtils.uniqueID()}")
 
-  val configMapNameDriver: String = configMapName(s"spark-drv-${KubernetesUtils.uniqueID()}")
+  def configMapNameDriver: String = configMapName(s"spark-drv-${KubernetesUtils.uniqueID()}")
```

Comment:
It seems that you have a different usage for this `KubernetesClientUtils` class. Actually, this is not supposed to be used like that initially.

Reply:
If we need to change this, I'm wondering `val configMapNameExecutor` is safe or not in the same reason.

Reply:
We did not encounter issue with this one. But I can dig in the code to try figuring out if it could also be changed